CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+## 1.2.0
+
+### Minor Changes
+
+* Add support for overriding the EncryptionContext in DynamoDBEncryptor #60
+* Update examples to use PUT instead of Clobber #60
```

Comment:
NP: I would consider all of the rest of these changes to be "maintenance", since they are not actually user-facing changes to the library (and in fact are not even changes to the library at all).

Reply:
It seems like it would make sense to include some changelog that describes roughly what changes we've made to the documentation, since it tags a rough date to when we started advising people to A instead of B. What do you think? I split it from minor changes to a documentation section, but I'm happy to remove it if we think it is clutter.

Reply:
Yeah, that's what I meant: just calling them something else in the changelog. "Documentation" works for me.